test/xpu/run_test_with_only.py

```diff
@@ -7,35 +7,53 @@
 
 def launch_test(test_case, skip_list=None, exe_list=None):
     if skip_list != None:
-        skip_options = " -k 'not " + skip_list[0]
+        skip_options = " -k \"not " + skip_list[0]
         for skip_case in skip_list[1:]:
             skip_option = " and not " + skip_case
             skip_options += skip_option
-        skip_options += "'"
-        test_command = (
-            "PYTORCH_ENABLE_XPU_FALLBACK=1 PYTORCH_TEST_WITH_SLOW=1 pytest -v "
-            + test_case
-        )
-        test_command += skip_options
-        return os.system(test_command)
+        skip_options += "\""
+        if os.name == "nt":
```

Comment:
Is it better to set `os.environ["PYTORCH_ENABLE_XPU_FALLBACK"] = 1` for both Windows and Linux?